llm_bioinformatics_research/README.md

```diff
@@ -77,30 +77,30 @@ Note: users must request database admin for access
 
```

Comment:
Overall your code is well organized and you've done a really nice job on the code refactoring and back button navigation. I've left one suggestion and two things that I think should be updated!